tests/testthat/test-linter_tags.R

```diff
@@ -0,0 +1,20 @@
+test_that("available_linters returns a data frame", {
+  avail <- available_linters()
+  avail2 <- available_linters(c("lintr", "not-a-package"))
+  empty <- available_linters("not-a-package")
+
+  expect_is(avail, "data.frame")
+  expect_identical(avail, avail2)
+  expect_equal(nrow(empty), 0L)
```

Comment:
After https://github.com/r-lib/lintr/pull/910 we should use `expect_identical()` by default:

> `expect_identical()` is the baseline; `expect_equal()` relaxes the test to ignore small numeric differences.

Reply:
changed